server/main.py

```
@@ -324,4 +324,4 @@ def on_fetch(request, env):
 
 if __name__ == "__main__":
     import uvicorn
-    uvicorn.run(app, host="0.0.0.0", port=8000)
\ No newline at end of file
+    uvicorn.run(app, host="0.0.0.0", port=8001)
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)